casr/src/bin/casr-afl.rs

```diff
@@ -75,6 +75,16 @@ fn main() -> Result<()> {
                 .value_parser(clap::value_parser!(PathBuf))
                 .help("Output directory with triaged reports")
         )
+        .arg(
+            Arg::new("seed")
```

Comment:
I think, we need to find an appropriate name for this option.
Seed is confusing.

In fuzzing  seed is input data that program processes normally.
Also, seed is used for random generators.

Please remove, short option too, only long is needed.

Reply:
Maybe, this way?

--update-clusters PREV_CRASHES_DIR

Update clusters in PREV_CRASHES_DIR with new crashes from INPUT_DIR and store accumulated crash report clusters to OUTPUT_DIR

CASR_PREV_CRASHES_DIR

Reply:
Why only the long version?

Reply:
> Maybe, this way?
> 
> --update-clusters PREV_CRASHES_DIR
> 
> Update clusters in PREV_CRASHES_DIR with new crashes from INPUT_DIR and store accumulated crash report clusters to OUTPUT_DIR
> 
> CASR_PREV_CRASHES_DIR

LGTM

Reply:
Renamed

Reply:
Renamed x2